docs/api/encoding.md

```diff
@@ -3,7 +3,9 @@
 An implementation of the Web standard encoding API.
 The Encoding API provides a mechanism for handling text in various character encodings, including legacy non-UTF-8 encodings.
 
-::: danger ⚠️ jstz's implementation is not fully spec compliant ⚠️ :::
+::: danger
+⚠️ `jstz`'s implementation is not fully spec compliant ⚠️
```

Comment:
Can we say why not or would the list of reasons be embarrassingly long.

Reply:
For encoding I think it is simply the fact that we don't support text encoder / decoder streams

Reply:
But in other cases, the list would be embarrasingly long 😬